.github/workflows/ci.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-12, macos-latest]
```

Comment:
For posterity: `macos-13` was funky, but this is still a huge improvement as it tests on Intel and M1 macOS